Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -73,13 +78,19 @@ extension StripeCardReaderService: CardReaderService {
         // Enough code to pass a test
         discoveryCancellable = Terminal.shared.discoverReaders(config, delegate: self, completion: { error in
             if let error = error {
-                print("discoverReaders failed: \(error)")
+                print("***** discoverReaders failed: \(error)")
```

Comment:
Glad I'm not the only person that does things like this :)

Reply:
rofl. It me. All the time.